glue_astronomy/translators/tests/test_regions.py

```diff
@@ -32,25 +33,37 @@ def setup_method(self, method):
         self.data.coords = WCS_CELESTIAL
         self.dc = DataCollection([self.data])
 
-    def test_rectangular_roi(self):
+    @pytest.fixture
+    def setup_rois(self):
+        # define different ROI shapes that are used in several tests.
 
-        subset_state = RoiSubsetState(self.data.pixel_component_ids[1],
-                                      self.data.pixel_component_ids[0],
-                                      RectangularROI(1, 3.5, -0.2, 3.3))
+        rois = {'CircularROI': RoiSubsetState(self.data.pixel_component_ids[1],
+                                              self.data.pixel_component_ids[0],
+                                              CircularROI(1, 3.5, 0.75)),
+                'RectangularROI': RoiSubsetState(self.data.pixel_component_ids[1],
+                                                 self.data.pixel_component_ids[0],
+                                                 RectangularROI(0, 2, 0, 7)),
```

Comment:
Any reason to change existing test input? I think it is good to test out of bounds coords.

```suggestion
                                                 RectangularROI(1, 3.5, -0.2, 3.3)),
```

Reply:
i wanted the centers of all subsets to be the same so parameterizing the test was simpler.